client/src/App/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import ThemeProvider from '../components/ThemeProvider';
+import {
+  Home,
+  Login,
+  Signup,
+  Logout,
+  Nutrition,
+  MyProfile,
+  Calories,
+  MyFood,
+  Food,
+  Exercise,
+} from '../Utils/constant';
+import HomePage from '../Pages/Home';
+import LoginPage from '../Pages/Login';
+import SignupPage from '../Pages/Signup';
+import LogoutPage from '../Pages/Logout';
+import NutritionPage from '../Pages/Nutrition';
+import MyProfilePage from '../Pages/Profile';
+import CaloriesPage from '../Pages/Calories';
+import MyFoodPage from '../Pages/MyFood';
+import FoodPage from '../Pages/Food';
+import ExercisePage from '../Pages/Exercise';
+import './style.css';
+
+function App() {
+  return (
+    <ThemeProvider>
+      <Router>
+        <Switch>
+          <Route exact path={Home} component={HomePage} />
+          <Route exact path={Login} component={LoginPage} />
+          <Route exact path={Signup} component={SignupPage} />
+          <Route exact path={Logout} component={LogoutPage} />
+          <Route exact path={Nutrition} component={NutritionPage} />
+          <Route exact path={MyProfile} component={MyProfilePage} />
+          <Route exact path={Calories} component={CaloriesPage} />
+          <Route exact path={MyFood} component={MyFoodPage} />
+          <Route exact path={Food} component={FoodPage} />
+          <Route exact path={Exercise} component={ExercisePage} />
+        </Switch>
```

Comment:
If the user enters a path that doesn't match any of these, no routes will match and they will see a blank screen.  You could make this a little more user-friendly by redirecting them to the home path if none of those paths match:

```
    ...
    <Route exact path={MyFood} component={MyFoodPage} />
    <Route exact path={Food} component={FoodPage} />
    <Route exact path={Exercise} component={ExercisePage} />
    <Redirect to='/' />
</Switch>
```

A `Switch` component will only render the first child which matches and will then stop checking the rest of the children, so by placing the `Redirect` component at the bottom of the children of `Switch`, it will always be handled if no prior routes match, but it will be ignored if any of the prior routes match. 
This way, if the user accidentally enters an incorrect path, they'll simply be redirected to Home.

Reply:
sure, I'll change that
thanks Tom for reminding me of this important point.